### PR TITLE
Add onboarding workflow for approval process with operations

### PR DESCRIPTION
**Pull request by masayag**
Signed-off-by: Moti Asayag <masayag@redhat.com>

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: onboarding/onboarding.sw.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+description: The Onboarding workflow demonstrates approval, operations placeholder and notifications
+dataInputSchema: schemas/onboarding__main-schema.json
+functions:
+  - name: jiraCreateIssue
+    operation: specs/jira-openapi.yaml#createIssue
+  - name: jiraGetIssue
+    operation: specs/jira-openapi.yaml#getIssue
+  - name: jiraCloseIssue
+    operation: specs/jira-openapi.yaml#transitionIssue
+  - name: jiraGetIssueTransitions
```
Comment (gabriel-farache) [inline onboarding/onboarding.sw.yaml:14]: this does not exists in the openapi spec file

File: onboarding/onboarding.sw.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+name: "Onboarding with approval and notifications"
+description: The Onboarding workflow demonstrates approval, operations placeholder and notifications
+dataInputSchema: schemas/onboarding__main-schema.json
+functions:
+  - name: jiraCreateIssue
+    operation: specs/jira-openapi.yaml#createIssue
+  - name: jiraGetIssue
+    operation: specs/jira-openapi.yaml#getIssue
+  - name: jiraCloseIssue
+    operation: specs/jira-openapi.yaml#transitionIssue
```
Comment (gabriel-farache) [inline onboarding/onboarding.sw.yaml:13]: this does not exists in the openapi spec file

**Comment by masayag** [comment]
I'm closing this one. Let's move it to serverless-workflows repository.

**Comment by masayag** [comment]
Created here: https://github.com/parodos-dev/serverless-workflows/pull/241

**Pull request closed without merging**